WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -566,22 +553,20 @@ public void onPostButtonClicked(int buttonType, PostModel postClicked) {
                 ActivityLauncher.viewPostPreviewForResult(getActivity(), mSite, post);
                 break;
             case PostListButton.BUTTON_STATS:
-                ActivityLauncher.viewStatsSinglePostDetails(getActivity(), mSite, post, mIsPage);
+                ActivityLauncher.viewStatsSinglePostDetails(getActivity(), mSite, post, false);
```

Comment:
Same here, this is only used by posts.

Reply:
Fixed in 78db0202950f3a865e8cb3f63872f9495134d43f.